src/test/java/com/squareup/javapoet/TypeSpecTest.java

```diff
@@ -2210,29 +2211,53 @@ private String toString(TypeSpec typeSpec) {
         + "}\n");
   }
 
-  @Test
-  public void initializerBlockUnsupportedExceptionOnInterface() {
+  @Test public void initializerBlockUnsupportedExceptionOnInterface() {
     TypeSpec.Builder interfaceBuilder = TypeSpec.interfaceBuilder("Taco");
-
     try {
       interfaceBuilder.addInitializerBlock(CodeBlock.builder().build());
       fail("Exception expected");
     } catch (UnsupportedOperationException e) {
     }
   }
 
-  @Test
-  public void initializerBlockUnsupportedExceptionOnAnnotation() {
-
+  @Test public void initializerBlockUnsupportedExceptionOnAnnotation() {
     TypeSpec.Builder annotationBuilder = TypeSpec.annotationBuilder("Taco");
-
     try {
       annotationBuilder.addInitializerBlock(CodeBlock.builder().build());
       fail("Exception expected");
     } catch (UnsupportedOperationException e) {
     }
   }
 
+  @Test public void lineWrapping() {
+    MethodSpec.Builder methodBuilder = MethodSpec.methodBuilder("call");
+    methodBuilder.addCode("$[call(");
+    for (int i = 0; i < 32; i++) {
+      methodBuilder.addParameter(String.class, "s" + i);
+      methodBuilder.addCode(i > 0 ? ",$W$S" : "$S", i);
```

Comment:
we probably want a helper that takes a `List` and gives you “comma separated” things